buildSrc/src/main/groovy/org/elasticsearch/gradle/plugin/PluginBuildPlugin.groovy

```diff
@@ -23,18 +23,19 @@ import org.elasticsearch.gradle.test.RestIntegTestTask
 import org.elasticsearch.gradle.test.RunTask
 import org.gradle.api.Project
 import org.gradle.api.Task
+import org.gradle.api.tasks.SourceSet
 import org.gradle.api.tasks.bundling.Zip
 
 /**
  * Encapsulates build configuration for an Elasticsearch plugin.
  */
-class PluginBuildPlugin extends BuildPlugin {
+public class PluginBuildPlugin extends BuildPlugin {
```

Comment:
I'm trying to make them "less groovy", but keep them groovy. This leaves us the option to use eg closures and string substitutions, which can be nice when needed. For access modifiers specifically, it was brought to my attention that people that are used to java looking at these files get confused because they think "these are all package protected". Personally I got used to just thinking "I know I'm in groovy so methods/classes default to public" (but members variables don't!), but if this helps others get into the code easier, then I don't mind being explicit with the access modifiers.